var/spack/repos/builtin/packages/py-pydicom/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPydicom(PythonPackage):
+    """Pure python package for DICOM medical file reading and writing
+
+    pydicom is a pure Python package for working with DICOM files. It lets you
+    read, modify and write DICOM data in an easy "pythonic" way."""
+
+    homepage = "https://github.com/pydicom/pydicom"
+    pypi     = "pydicom/pydicom-2.1.2.tar.gz"
+
+    version('2.1.2', sha256='65f36820c5fec24b4e7ca45b7dae93e054ed269d55f92681863d39d30459e2fd')
+
+    depends_on('python@3.6.1:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+
+    # py-numpy is not required but is needed for Pixel Data
+    depends_on('py-numpy',        type=('build', 'run'))
```

Comment:
Should we add a variant for this?

Reply:
I did think about that. My thinking is that numpy is generally useful and pretty common so might as well include it. But I can see it as a variant as well. That is kind of why I separated it out and commented on it. I guess that should be a run dependency though, not a build and run type.

Reply:
After more thought, I think it makes sense to have a variant for this so I will add that.